docs/week14_scrum.md

```diff
@@ -0,0 +1,34 @@
+# Team Meetings
+
+Date: Jan 14, 2022
+
+Attendees: Raouf Tiliouine, Josh Cooper, Brian Zhou, Iwan Levin
+
+Absent:
+
+## 1. Project
+
+**Progress**
+
+Our first client meeting of the semester is scheduled for next week so we've decided to delay development work until then.
+
+**Blockers + Surprises**
+
+N/A
+
+## 2. Learning Activity
+
+N/A
+
+## 3. Plan
+
+**Task for this week:**
+
+- Fix malformed buffer copy
```

Comment:
according to the template this should have a name associated

Reply:
Added